monitoring/dashboard.py

```diff
@@ -525,6 +542,111 @@ def top10_errors_by_bucket(title, code):
+    dataSource="${DS_PROMETHEUS}",
+    calc="mean",
+    format=UNITS.PERCENT_FORMAT,
+    min=0,
+    max=100,
+    noValue="-",
+    targets=[Target(
+        expr="\n".join([
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state="healthy"}[$__rate_interval])) * 100',  # noqa: E501
+            "   /",
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state=~"^(unhealthy|healthy)$"}[$__rate_interval]))',  # noqa: E501
+        ]),
+        legendFormat="Online rate",
+    )],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("red",     1, 0.0),
+        Threshold("orange",  2, 90.0),
+        Threshold("green",   3, 95.0),
+    ],
+)
+
+
+quotaStatusCode = TimeSeries(
+    title="Quota evaluation status code over time",
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=30,
+    lineInterpolation="smooth",
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum by (code) (rate(s3_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"2.."}[$__rate_interval]))',  # noqa: E501
+        legendFormat="Success",
+    ), Target(
+        expr='sum by (code) (rate(s3_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"4.."}[$__rate_interval]))',  # noqa: E501
+        legendFormat="Quota Exceeded",
+    )],
+)
+
+quotaByAction = TimeSeries(
+    title="Quota evaluaton rate per S3 action",
+    dataSource="${DS_PROMETHEUS}",
+    legendDisplayMode="table",
+    legendPlacement="right",
+    legendValues=["min", "mean", "max"],
+    lineInterpolation="smooth",
+    unit=UNITS.OPS_PER_SEC,
+    targets=[
+        Target(
+            expr='sum(rate(s3_quota_evaluation_duration_seconds_count{namespace="${namespace}"}[$__rate_interval])) by(action)',  # noqa: E501
+            legendFormat="{{action}}",
+        )
+    ]
+)
+
+
+averageQuotaLatencies = TimeSeries(
+    title="Average quota evaluation latencies",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="smooth",
+    spanNulls=3*60*1000,
+    unit=UNITS.SECONDS,
+    targets=[
+        average_quota_latency_target(
+            title="Bucket quota (success)", type='"bucket"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Bucket quota exceeded", type='"bucket"', code='~"4.."'),
+        average_quota_latency_target(
+            title="Account quota (success)", type='"account"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Account quota exceeded", type='"account"', code='~"4.."'),
+        average_quota_latency_target(
+            title="Account and Bucket quota (success)", type='"bucketAccount"',
+            code='~"2.."'),
+        average_quota_latency_target(
+            title="Account quota exceeded", type='"bucketAccount"',
+            code='~"4.."'),
+        average_quota_latency_target(
+            title="Quota exceeded (delete data)", type='"delete"'),
```

Comment:
`data` is a little vague, is it all delete action ?
Also is the delete included in the above quota on account and bucket ?

Reply:
This is indeed for the delete APIs. I can rename it "Data deletion".
This metric is not part of the above values, it's a special case when you delete data, the other ones track the writes.

Reply:
maybe you should describe that as well for other metrics

Reply:
I reworked the timeserie, now we say if we write or delete, with some details:

![image](https://github.com/scality/cloudserver/assets/91462779/eaf4c4b7-9a35-4f7b-8294-18e552b39acd)

Reply:
for `Data write (bucket)` you could add that it's for success so we don't think it encapsulate the exceeded also